### PR TITLE
Adjustment of the ECIP process

### PR DESCRIPTION
**Pull request by TokenHash**
• Structuring the documentation to include : ECIP Home (this summary), connections the formal ECIP process (ECIP-1000), the proper ECIP-Sample doc, and the proper ECIP-markdown-template when appropriate.
• Added  a 'Summary' section with links to the ECIP Process doc to better introduce ECIPs to newcomers.
• Added a 'Types of Participants' section to clarify who, responsibilities,  and how each type of participant contributes to the ECIP process. Also to make sure 'Authors" understand their role.
• Made spelling and grammatical corrections for clarity.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,26 @@
...
+
+# Types of Participants
+
+As you will see by reading this summary and the other documents suggested above, there are several parties that participate in the life cycle of an ECIP:
+
+- **ECIP Author:** This is you! The author is a person interested in imporving ETC, therefore responsible for learning how the ECIP process works, so he or she can effectively propose protocol changes, informational, or procedural improvements. The author is also reponsible for following up on the ECIP life cycle throught all the status steps, from 'draft' to 'active' as described below, so the proposals become a reaity. If not, it is possible that the ECIP will be eventually rejected.
+- **Editors:** Editors are individuals, mainly ETC volunteers, who are members of the Ethereum Classic Github Organization responsible for assigning numbers to ECIPs, merging them, listing them in the ECIP directory, checking if they are ready, sound and complete, if the status steps are followed, and their motivation, accuracy and proper licensing terms, amongst other tasks.
+- **Core Developer Team:** Aa
```
Comment (zmitton) [inline README.md]: there is not a special one team
```suggestion
- **Developer Teams:** Aa
```

**Comment by zmitton** [comment]
seem like useful clarification to me 👍

**Comment by TokenHash** [comment]
thanks, I had closed this pr. Can you check out https://github.com/ethereumclassic/ECIPs/pull/88 instead?

**Comment by TokenHash** [comment]
I will add that change tho

**Pull request closed without merging**